src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -275,6 +283,27 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     executeOrResume(backendCall, runIdForResumption = runId)
   }
 
+  def useCachedCall(cachedCall: BackendCall, backendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = Future {
+    val log = workflowLoggerWithCall(backendCall)
+    authenticateAsUser(backendCall.workflowDescriptor) { interface =>
+      Try(interface.copy(cachedCall.callGcsPath, backendCall.callGcsPath)) match {
+        case Failure(ex) =>
+          log.error(s"Exception occurred while attempting to copy outputs from ${cachedCall.callGcsPath} to ${backendCall.callGcsPath}", ex)
+          FailedExecutionHandle(ex)
+        case Success(_) => postProcess(backendCall) match {
+          case Success(outputs) => SuccessfulExecutionHandle(outputs, backendCall.downloadRcFile.get.stripLineEnd.toInt, backendCall.hash)
+          case Failure(ex: AggregatedException[_]) if ex.exceptions.map(_.exception).exists(_.isInstanceOf[SocketTimeoutException]) =>
```

Comment:
Why does `TryUtil` box exceptions into `Failure`s?  Without the box this code could be simpler and not have to resort to `instanceOf`.